docs/resources/pipeline.md

```diff
@@ -76,13 +76,26 @@ The following arguments are supported:
 * `library` blocks - Specifies pipeline code and required artifacts. Syntax resembles [library](cluster.md#library-configuration-block) configuration block with the addition of a special `notebook` & `file` library types that should have the `path` attribute. *Right now only the `notebook` & `file` types are supported.*
 * `cluster` blocks - [Clusters](cluster.md) to run the pipeline. If none is specified, pipelines will automatically select a default cluster configuration for the pipeline. *Please note that DLT pipeline clusters are supporting only subset of attributes as described in [documentation](https://docs.databricks.com/data-engineering/delta-live-tables/delta-live-tables-api-guide.html#pipelinesnewcluster).*  Also, note that `autoscale` block is extended with the `mode` parameter that controls the autoscaling algorithm (possible values are `ENHANCED` for new, enhanced autoscaling algorithm, or `LEGACY` for old algorithm).
 * `continuous` - A flag indicating whether to run the pipeline continuously. The default value is `false`.
-* `development` - A flag indicating whether to run the pipeline in development mode. The default value is `true`.
+* `development` - A flag indicating whether to run the pipeline in development mode. The default value is `false`.
```

Comment:
Note for the reader or this PR, this was incorrect

https://openapi.dev.databricks.com/api/workspace/pipelines/create